src/librustc/hir/intravisit.rs

```diff
@@ -814,6 +861,7 @@ pub fn walk_arm<'v, V: Visitor<'v>>(visitor: &mut V, arm: &'v Arm) {
 }
 
 pub fn walk_vis<'v, V: Visitor<'v>>(visitor: &mut V, vis: &'v Visibility) {
+    // FIXME(mw): What is this NodeId in there? And should it be visited?
```

Comment:
The path in visibility `pub(path)` has a resolution in `def_map` keyed by this id.